### DISCUSSION TITLE
How to pass data from a hidden input

### DISCUSSION THREAD

**Discussion by willmkt** [livewire/livewire#1331] [4. General]
Hi i'm trying to pass data from a hidden input from using wire:model on submit, but it returns null.

<input type="hidden" name="total" value="{{ $total }}" wire:model="total">

Is there other way to pass this data?

**Comment by mblarsen** [comment]
Can you show your form and hidden input code. Then people will be able to give you the best help.

See [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).

**Reply by willmkt** [reply]
I forgot to put the example

<input type="hidden" name="total" value="{{ $total }}" wire:model="total">

    public function saveEstimate($request)
    {
        //dd(request('totals'));
        $estimate = Estimate::create([
                'construction_id' => $this->construction,
                'segment_id' => $this->segment,
                estimate_total' => $this->total
        ]);

}

When i submit it returns null

**Comment by asantibanez** [comment]
I believe I've done this before without `wire:model` but using `value={{ $value }}` in the hidden input.

**Reply by KalanaPerera** [reply]
not working :(

**Comment by willmkt** [comment]
Already solved by passing $this->total to mount..

Thanks.
